NewForce-Cohort-6/tabloid-cli-coven-of-the-lunar-owls
# Jg list tags#21

# Description

Ticket #21 Lists all the tags that have been added to the Tag table of the database.

# Testing Instructions

1. Pull down the branch jg-ListTags#21 in the usual way. 
2. Navigate to the Tag Management menu from the Main Menu
3.  Select List Tags.   
4. Note if a list of the tag names is displayed. If so, this feature works.



# Checklist:
